var/spack/repos/builtin/packages/py-haphpipe/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHaphpipe(PythonPackage):
+    """HAplotype and PHylodynamics pipeline for viral assembly,
+    population genetics, and phylodynamics."""
+
+    homepage = "https://github.com/gwcbi/haphpipe"
+    url      = "https://github.com/gwcbi/haphpipe/archive/v1.0.3.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('1.0.3', sha256='9a9e8632a217ff4207c1dea66887a471e0ea04bbb7c0f0d72631acaba214bd37')
+
+    # Deps. taken from
+    # https://github.com/bioconda/bioconda-recipes/blob/master/recipes/haphpipe/meta.yaml
+    depends_on('python@3.7:', type=('build', 'run'))
+    depends_on('py-future', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
+    depends_on('py-biopython@1.73:', type=('build', 'run'))
```

Comment:
Where does this version restriction come from?

Reply:
I basically copied the deps from: http://bioconda.github.io/recipes/haphpipe/README.html